src/views/Feed/FeedExtra.js

```diff
@@ -14,7 +15,17 @@ function FeedExtra(props) {
   )
   const rest = getUnhandledProps(FeedExtra, props)
 
-  return <div {...rest} className={classes}>{children || text}</div>
+  if (!images || _.isBoolean(images)) {
+    return <div {...rest} className={classes}>{children || text}</div>
+  }
+
+  const imagesJSX = images.map((image, index) => {
+    const key = `${index}`
```

Comment:
Hm, I see.

/cc @davezuko we we're just talking about `weak-key`, here we have an array of objects with possibly no unique key/value pairs.  Ideas for generating unique React keys?